runtime/tests/automl_runtime/forecast/pmdarima/model_test.py

```diff
@@ -147,16 +147,16 @@ class TestMultiSeriesArimaModel(unittest.TestCase):
     def setUp(self) -> None:
         num_rows = 9
         self.X = pd.concat([
-            pd.to_datetime(pd.Series(range(num_rows), name="date").apply(lambda i: f"2020-10-{i + 1}")),
+            pd.to_datetime(pd.Series(range(num_rows), name="date").apply(lambda i: f"2020-{i + 1:02d}-13")),
             pd.Series(range(num_rows), name="y")
         ], axis=1)
         model = ARIMA(order=(2, 0, 2), suppress_warnings=True)
         model.fit(self.X.set_index("date"))
         pickled_model = pickle.dumps(model)
         pickled_model_dict = {"1": pickled_model, "2": pickled_model}
-        start_ds_dict = {"1": pd.Timestamp("2020-10-01"), "2": pd.Timestamp("2020-10-01")}
-        end_ds_dict = {"1": pd.Timestamp("2020-10-09"), "2": pd.Timestamp("2020-10-09")}
-        self.arima_model = MultiSeriesArimaModel(pickled_model_dict, horizon=1, frequency='d',
```

Comment:
In the unit tests can we cover both day and month frequency? I felt the coverage is necessary because daily data could be very different from monthly data in terms of forecasting